### ISSUE TITLE
Bazel warning regarding deprecated `@@bazel_tools//src/conditions:host_windows_x64_constraint` being used by `oci_image`

### ISSUE THREAD

**Issue by fishy** [bazel-contrib/rules_oci#819]
We recently noticed this warning from bazel:

```
WARNING: /path/to/BUILD.bazel:82:10: in _write_file rule //:docker_write_cmd: target '//:docker_write_cmd' depends on deprecated target '@@bazel_tools//src/conditions:host_windows_x64_constraint': No longer used by Bazel and will be removed in the future. Migrate to toolchains or define your own version of this setting.
```

the line in `BUILD.bazel` is an `oci_image` rule:

```bzl
load("@rules_oci//oci:defs.bzl", "oci_image")

...

oci_image(  # <- this is line 82
    name = "docker",
    base = BASE_IMAGE,
    cmd = [
        "/go/bin/foo",
    ],
    tars = [
        ":bin",
    ],
    workdir = WORKDIR,
)

...
```

We are using rules_oci 2.2.6.
